pysrc/pip_resolve.py

```diff
@@ -199,9 +199,21 @@ def get_requirements_list(requirements_file_path, dev_deps=False):
         with io.open(requirements_file_path, 'r', encoding='utf-8') as f:
             requirements_data = f.read()
         parsed_reqs = pipfile.parse(requirements_data)
-        req_list = list(parsed_reqs.get('packages', []))
+        req_list = list(parsed_reqs.get('packages') or [])
         if dev_deps:
-            req_list.extend(parsed_reqs.get('dev-packages', []))
+            req_list.extend(parsed_reqs.get('dev-packages') or [])
+        
+        # Error informatively if there are no dependencies
+        if req_list == []:
+            if parsed_reqs['dev-packages'].__len__() > 0 and dev_deps == False:
```

Comment:
`parsed_reqs['dev-packages'].__len__()` might throw `AttributeError: 'NoneType' object has no attribute '__len__'` in case if `dev_packages` is `None`
Other way to do this if statement might be like this: `if parsed_reqs.get('dev-packages') and not dev_deps:`

Reply:
Looks much more idiomatic too. Thanks :)